cluster/iam_ssm_document.tf

```
@@ -127,9 +127,9 @@ data "aws_iam_policy_document" "automation_ssm_ec2" {
   }
 
   statement {
-    sid       = "RunCommand"
+    sid       = "SendCommand"
     effect    = "Allow"
-    actions   = ["ssm:RunCommand"]
+    actions   = ["ssm:SendCommand"]
     resources = ["arn:aws:ssm:*:*:document/*"]
```

Per the [service authorization reference](https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssystemsmanager.html), this should be widened so that `SendCommand` can apply to both documents and instances:

```suggestion
    resources = ["*"]
```